force-app/main/domain/LegacyHouseholds.cls

```diff
@@ -32,7 +32,7 @@
 * @date 2021
 * @description Domain class for Households in NPSP.
 */
-public inherited sharing class Households extends fflib_SObjects {
+public inherited sharing class LegacyHouseholds extends fflib_SObjects {
```

Comment:
Does this refer to the actual Household sObject? I like the rename since it does a good job of differentiating from Household Accounts, we should just agree on the terminology.

Reply:
Yep!  The proposed changes in thes PR:

<img width="650" alt="2021-08-07_15-47-55" src="https://user-images.githubusercontent.com/46458081/128615673-a40f07a0-4a88-4337-a128-1611fd0182d9.png">